src/System.Net.Mail/tests/Functional/AttachmentTest.cs

```diff
@@ -32,6 +32,23 @@ public void ConstructorNullName()
             Assert.Equal(null, attach.Name);
         }
 
+        [Fact]
+        public void ConstructorPathName()
+        {
+            string attachFile = Path.GetTempFileName();
+            try
+            {
+                var fileStream = File.Create(attachFile);
+                fileStream.Close();
+                Attachment attach = new Attachment(attachFile);
```

Comment:
you're probably gonna have to wrap everything in using statement as I suspect that's what's causing the failures

Reply:
ok